engine/src/main/scala/cromwell/util/PromiseActor.scala

```diff
@@ -0,0 +1,35 @@
+package cromwell.util
+
+import akka.actor._
+
+import scala.concurrent.{Future, Promise}
+
+object PromiseActor {
+  /**
+    * Sends a message to an actor and returns the future associated with the fullfilment of the reply
+    * Can be used instead of the akka `ask` semantics, without any timeout
+    * @param sendTo The actor that is wished to be asked something
+    * @param message The message to send to the actor
+    * @param actorSystem ActorSystem from which to create the actor from
+    * @return The future associated with the answer to the `ask`
+    */
+  def askNoWait(message: Any, sendTo: ActorRef)(implicit actorSystem: ActorSystem): Future[Any] = {
```

Comment:
This name feels off, both because `ask` reflexively raises my hackles and because `NoWait` makes it sound like fire-and-forget which is actually the opposite of what it does.